garage/tf/models/base.py

```diff
@@ -0,0 +1,84 @@
+"""This file contains the abstraction class for models."""
+from garage.tf.models import AutoPickable
+
+
+class BaseModel:
```

Comment:
I don't see BaseModel is used anywhere. When will we use it?

Reply:
When people don't want `AutoPickable` but want to use the API from `Model`.

Reply:
How does the user pickle model if he uses BaseModel?

Reply:
Yeah, I think this is good for the same reasons expressed above.